src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketClientFactoryImpl.java

```diff
@@ -82,27 +85,29 @@ public BitbucketProjectSearchClient getProjectSearchClient() {
         return new BitbucketProjectSearchClient() {
 
             @Override
-            public BitbucketPage<BitbucketProject> get(@CheckForNull String name) {
+            public Stream<BitbucketPage<BitbucketProject>> get(@CheckForNull String name) {
```

Comment:
I don't think we need streaming support for the search endpoint, we need only the first page. When we implement paging for search endpoints, it will take a `start` and `limit` query parameter which would be passed to the Bitbucket server

Reply:
I have reverted the changes.